File: Search Algorithms/Ruby/LinearSearch.rb
def linear_search(array, key)
  if array.index(key).nil?
    return -1
  else
    return "#{key} at index #{array.index(key)}"
  end
end


arr = [7, 6, 25, 19, 8, 14, 3, 16, 2, 0]
key = 3
p linear_search(arr, key)

File: Search Algorithms/Kotlin/LinearSearch.kt
fun linearSearch(arr: IntArray, key: Int) {
    for (i in arr.indices) if (arr[i] == key) {
        println("$key found at position $i")
        return
    }
    println("$key not found")
}

fun main(args: Array<String>) {
    val arr = intArrayOf(9, 3, 1, 4, 5)
    linearSearch(arr, 4)
    linearSearch(arr, 2)
}

